src/components/tx/security/tenderly/index.tsx

```diff
@@ -43,13 +43,15 @@ const TxSimulationBlock = ({ transactions, disabled, gasLimit }: TxSimulationPro
   const isDarkMode = useDarkMode()
   const { safeTx } = useContext(SafeTxContext)
   const {
-    simulation: { simulateTransaction, simulationRequestStatus, resetSimulation },
+    simulation: { simulation, simulateTransaction, simulationRequestStatus, resetSimulation },
   } = useContext(TxInfoContext)
 
   const isLoading = simulationRequestStatus === FETCH_STATUS.LOADING
   const isSuccess = simulationRequestStatus === FETCH_STATUS.SUCCESS
   const isError = simulationRequestStatus === FETCH_STATUS.ERROR
 
+  const isCallTraceError = isSuccess && getCallTraceErrors(simulation).length > 0
```

Comment:
The `simulation.status` [also needs to be taken into account](https://github.com/safe-global/safe-wallet-web/pull/2206/files).

For some reason merging the epic didn't include this fix.

Reply:
Good find! We also extracted the status logic into the context